desktop/cmp/rest/RestGridModel.js

```diff
@@ -157,8 +156,8 @@ export class RestGridModel {
     }
 
     /** Load the underlying store. */
-    doLoadAsync(loadSpec) {
-        return this.store.loadAsync(loadSpec);
+    async doLoadAsync(loadSpec) {
+        return this.store.loadSupport.loadAsync(loadSpec);
```

Comment:
Another reference we could update if we wished to declare `loadAsync()` on `UrlStore`